installer/x86_64/install.sh

```diff
@@ -106,6 +106,11 @@ fi
 if [ "$install_env" != "build" ]; then
     onie_dev=$(blkid | grep ONIE-BOOT | head -n 1 | awk '{print $1}' |  sed -e 's/:.*$//')
     blk_dev=$(echo $onie_dev | sed -e 's/[1-9][0-9]*$//' | sed -e 's/\([0-9]\)\(p\)/\1/')
+
+    blk_suffix=
+    echo $blk_dev | grep -q mmcblk && blk_suffix="p"
+    echo $blk_dev | grep -q nvme0 && blk_suffix="p"
```

Comment:
I do not really understand the code intention here. Please have a detailed description on what you are trying to achieve here.

Reply:
Hi Sir,
We will echo blk_dev value and check if nvme0 is existed. (echo $blk_dev | grep -q nvme0 &&)
If  nvme0 is existed, do blk_suffix="p"

Actually This logic come from ONIE: https://github.com/opencomputeproject/onie/commit/ed5108d023d23a876767f0311a8bc43d67d34bd6

Thanks a lot